### PR TITLE
feat: add relayer key management support

### PR DESCRIPTION
**Pull request by aleksuss**
The PR adds support for managing relayer keys. Added the following new transactions: 
- `set_key_manager`
- `add_relayer_key`
- `remove_relayer_key`

### CODE REVIEW

File: src/cli/simple/command/mod.rs [hunk 1]
```diff
@@ -490,6 +508,56 @@ pub async fn paused_precompiles(client: Client) -> anyhow::Result<()> {
...
+    let message = manager.as_ref().map_or_else(
+        || "has been removed".to_string(),
+        |account_id| format!("{account_id} has been set"),
+    );
+    // TODO: Use RelayerKeyManagerArgs from engine-types instead.
+    let args = serde_json::to_vec(&serde_json::json!({ "key_manager": manager }))?;
+
+    contract_call!(
+        "set_key_manager",
+        "The key manager {message} successfully",
```
Comment (mandreyel) [inline src/cli/simple/command/mod.rs:522]:
```suggestion
        "The key manager {message} has been sent successfully",
```

File: src/cli/simple/mod.rs [hunk 2]
```diff
@@ -190,6 +192,13 @@ pub enum Command {
         #[arg(long)]
         seed: Option<u64>,
     },
+    /// Return randomly generated NEAR key for AccountId
+    GenerateNearKey {
+        /// AccountId
+        account_id: String,
```
[the remark this answers is not in the document]
Comment (aleksuss) [reply src/cli/simple/mod.rs:198]: There is no need for that. We just transition the account_id to a JSON object.

File: src/cli/simple/command/mod.rs (hunk 1 above)
Comment (aleksuss) [reply src/cli/simple/command/mod.rs:523]: This part is added above.

File: src/cli/simple/command/mod.rs [hunk 3]
```diff
@@ -490,6 +508,56 @@ pub async fn paused_precompiles(client: Client) -> anyhow::Result<()> {
     get_value::<u32>(client, "paused_precompiles", None).await
 }
 
+/// Set relayer key manager.
+pub async fn set_key_manager(client: Client, manager: Option<AccountId>) -> anyhow::Result<()> {
+    let message = manager.as_ref().map_or_else(
+        || "has been removed".to_string(),
+        |account_id| format!("{account_id} has been set"),
+    );
+    // TODO: Use RelayerKeyManagerArgs from engine-types instead.
```
[the remark this answers is not in the document]
Comment (aleksuss) [reply src/cli/simple/command/mod.rs:518]: Yes

**Review by mrLSD — APPROVED** [review]

**Pull request merged**